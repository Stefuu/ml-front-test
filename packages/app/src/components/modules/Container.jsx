import React from 'react'
import Box from '../elements/Box'

const Container = (props) => (
  <Box
    m='0 auto'
    maxWidth={1040}
    {...props}>
    {props.children}
  </Box>
)

export default Container
