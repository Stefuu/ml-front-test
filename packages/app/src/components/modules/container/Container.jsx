import React from 'react'
import Box from '../../elements/Box'

const Container = (props) => (
  <Box
    m='0 auto'
    maxWidth={props.maxWidth || 980}
    {...props}>
    {props.children}
  </Box>
)

export default Container
