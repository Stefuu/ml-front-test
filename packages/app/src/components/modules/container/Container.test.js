import React from 'react'
import { render } from 'react-testing-library'
import Container from './Container'

describe('Container', () => {
  it('should render [snapshot]', () => {
    const { container } = render(<Container />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
