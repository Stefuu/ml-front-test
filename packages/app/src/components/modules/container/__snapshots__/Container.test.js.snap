// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render [snapshot] 1`] = `
<div
  class="sc-bdVaJa fEwoCY"
/>
`;
